pkg/cloudprovider/suite_test.go

```diff
@@ -550,20 +412,56 @@ var _ = Describe("CloudProvider", func() {
 			awsEnv.EC2API.DescribeInstancesBehavior.Output.Set(&ec2.DescribeInstancesOutput{
 				Reservations: []*ec2.Reservation{{Instances: []*ec2.Instance{}}},
 			})
-			machine := coretest.Machine(v1alpha5.Machine{
```

Comment:
Just validating: we are removing this since the top-level `BeforeEach` is already creating this?

Reply:
Yup!